WebRTC~/Packages/com.unity.webrtc/Tests

```diff
@@ -0,0 +1 @@
+../../../Tests
```

Comment:
I am not sure this file is needed

Reply:
I think we need this in order to have the tests inside the test project.
There is actually a better way to do this though, and I will fix this soon